packages/@stylexjs/cli/src/transform.js

```diff
@@ -81,7 +81,7 @@ export async function compileDirectory(
 
   const compiledCSS = await styleXPlugin.processStylexRules(
     Array.from(config.state.styleXRules.values()).flat(),
-    config.useCSSLayers,
+    config.useCSSLayers !== 'none',
```

Comment:
This arg should be changed to an options object, and here we'd pass `{ layers }`